pkg/asset/internal/templates.go

```diff
@@ -806,13 +806,16 @@ data:
     .:53 {
         errors
         health
-        kubernetes cluster.local {{ .ServiceCIDR }} {
+        ready
```

Comment:
Do we need to update the liveness and readiness probes to make use of this?

Reply:
isnt it better to have that in a separate PR ?